Assets/MXR.SDK/Runtime/Types/ContentTypes.cs

```diff
@@ -1,42 +1,135 @@
 ﻿using System.Collections.Generic;
+using Newtonsoft.Json;
 
 namespace MXR.SDK {
+    /// <summary>
+    /// Base class for different kinds of deployable content supported
+    /// </summary>
     [System.Serializable]
     public class Content {
+        /// <summary>
+        /// Unique identifier of the content
+        /// </summary>
         public string id;
+
+        /// <summary>
+        /// Title of the content
+        /// </summary>
         public string title;
+
+        /// <summary>
+        /// Categories associated with the content
+        /// </summary>
         public List<string> categories = new List<string>();
+
+        /// <summary>
+        /// Description of the content
+        /// </summary>
         public string description;
+
+        /// <summary>
+        /// Whether the content has been configured to be hidden
+        /// on the ManageXR web dashboard. To be used by the
+        /// home screen implementing the SDK
+        /// </summary>
         public bool hidden;
+
+        /// <summary>
+        /// Local path of the icon. This may or may not exist.
+        /// Admin app downloads the icons to local storage.
+        /// Begins with /MightyImmersion. <see cref="MXRStorage.GetFullPath(string)"/>
+        /// can be used to get the full path
+        /// </summary>
         public string iconPath;
+
+        /// <summary>
+        /// URL of the remotely stored icon file
+        /// </summary>
         public string iconUrl;
+
+        /// <summary>
+        /// The sort order configured on the ManageXR web dashboard for this content.
+        /// To be used by the home screen implementing the SDK
+        /// </summary>
         public int sortOrder;
+
+        /// <summary>
+        /// ID of the ManageXR organization this device is set up for
+        /// </summary>
         public string organizationId;
 
+        /// <summary>
+        /// Whether this content has been configured to not launch when it's
+        /// being updated. To be used by the home screen implementing the SDK
+        /// </summary>
         public bool blockLaunchWhileUpdating = false;
-        
+
+        /// <summary>
```

Comment:
Should this be an ENUM, since there is a known set of values?

Reply:
Makes sense, can we change `mandatory` to "Mandatory" (M upper case) or "MANDATORY" instead of all lowercase? Same with `optional`

Reply:
Sure, good call - let's do MANDATORY, which is the style we do elsewhere. 

As long as this doesn't require a migration / admin app changes? If so, lets leave it as is.

Reply:
Content.Requirement enum introduced in d4652042ce39fbcf923de74d251335fd5e86fef5